public/pages/login.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style/style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Find My Musician.com</title>
+</head>
+
+<body>
+    <header>
+        <div class="container">
+        <h1>Find My Musician.com </h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="email-login">Log In</a>
+                </li>
+                <li>
+                    <a href="password-login">Sign Me Up</a>
+                </li>
+            </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="content">
+       
+            
+            <div class="even">
+                
+                <p class="history"> Find A Musician is a website dedicated to connecting musicians with other 
+                    like-minded musicians for collaboration and project assistance. Our platform allows musicians 
+                    to easily create an account and make posts asking for a helping hand or collaboration from other musicians.
+                    Whether you're looking for a guitarist for your band or a vocalist for your newest song our site offers a 
+                    user-friendly interface to help you find the right fit. Join our community of passionate and creative musicians 
+                    today and take your musical projects to the next level. 
+                                                                                       
+                </p>
+            </div>
```

Comment:
All of this content is for the home page. This page just needs two <input = "text"> for the password and username so the user can log in to their account. Look at page 9 of the wireframe for a visual.